homeassistant/components/notify/pushover.py

```diff
@@ -63,13 +63,21 @@ def send_message(self, message='', **kwargs):
 
         targets = kwargs.get(ATTR_TARGET)
```

Comment:
If targets might be `None`, another approach to ensure the `for` loop below works would be to give an empty list here:

`targets = kwargs.get(ATTR_TARGET, [])`

`